phoenix-tracing-webapp/src/build/build.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Do we still need this ant file? We're not building a war anymore, it's an embedded service. I think the whole ant step can be removed.